elastic/tests/runners/resources/ilm/nested/dont-read-me.json

```diff
@@ -0,0 +1,3 @@
+{
+  "policy": {}
```

Comment:
I didn't have to change the test itself, but adding this file cause the existing ilm test to fail, then succeed after adding the above fix :)